content/applications/inventory_and_mrp/barcode/setup/hardware.rst

```diff
@@ -76,7 +76,7 @@ Zebra scanner
 When using Zebra scanners, ensure the following keystroke configurations are set to prevent errors.
 
 Begin on the Zebra scanner's home screen, and select the :guilabel:`DataWedge` app (the app is
-represented by a :guilabel:`(light blue barcode)` icon).
+represented by a :guilabel:`light blue barcode` icon).
```

Comment:
By @samueljlieber's logic this should be wrapped in parenthesis, as well, since, theoretically, an emoji would normally accompany the label (if there were one available for us to use).

```suggestion
represented by a :guilabel:`(light blue barcode)` icon).
```

Reply:
Suggestion implemented ✅